packages/core/src/sass/app/_common/components/banner-dtrader-v2.scss

```diff
@@ -0,0 +1,45 @@
+.banner {
+    border-radius: var(--core-borderRadius-400);
+    padding: var(--core-spacing-800);
+    background-color: var(--core-color-solid-slate-700);
+    height: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: var(--core-spacing-800);
+
+    &__container {
+        width: 100%;
+        height: var(--core-size-4000);
+        padding: var(--core-spacing-400);
+    }
+
+    &__currency {
+        width: var(--core-size-4000);
+        white-space: nowrap;
+
+        &--is-eu {
+            width: var(--core-size-2800);
+        }
+
+        .icon {
+            &-2 {
+                position: relative;
+                right: var(--temp-static-spacing-7);
+            }
+            &-3 {
+                position: relative;
+                right: calc(2 * var(--temp-static-spacing-7));
+            }
+        }
+    }
+
+    &__text:is(p) {
```

Comment:
```suggestion
    &__text {
```
A question: Why do we need to specify the tag name? will it work without `:is(p)`? :)
Same for `&__balance:is(h5)` in **packages/core/src/sass/app/_common/layout/header.scss**: Why do we need to specify `:is(h5)`? )

Reply:
No, it will not work:(
I need to increase the weight of css selector in order to rewrite the style from quill. Unfortunately, the only official way to control the color is to pass it via props (but it has only 4 allowed colors). The power of classNmae is not working without increasing the weight

<img width="505" alt="Screenshot 2024-06-24 at 9 16 45 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/1383f616-ed1d-4ae9-930f-531348d943ae">
<img width="505" alt="Screenshot 2024-06-24 at 9 16 58 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/2c2f53f9-b936-4bf6-a629-dda52bac6553">
![Screenshot 2024-06-24 at 9 21 37 AM](https://github.com/binary-com/deriv-app/assets/121025168/b0bf7a57-2674-4e87-bca7-f6811f70890d)

Reply:
![Screenshot 2024-06-24 at 9 21 37 AM](https://github.com/binary-com/deriv-app/assets/121025168/3d78c2b3-f7fb-488d-8c90-a7b2d6b313e5)